### DISCUSSION TITLE
Cannot add new taxa, with fix.

### DISCUSSION THREAD

**Discussion by foozleface** [Symbiota/community-board#43] [Problems/Bugs]
Hitting  /taxa/taxonomy/taxonomyloader.php and trying to add any new taxa, you'll get a pop up warning: "Parent identifier is not set! Make sure to select parent taxon from the list".

  The problem is setParent in taxa.taxonomyloader.js:

`
function setParent(parentName, unitind1){

	$.ajax({

		type: "POST",

		url: "rpc/gettid.php",

		async: true,

		data: { sciname: parentName }

	}).done(function( msg ) {

`

  This file is missing. Re-adding it from an older version (to taxa/taxonomy/rpc/gettid.php) solves the issue.

**Comment by foozleface** [comment]
Bad git merge on my part, files are NOT missing.

**Reply by arbolitoloco** [reply]
@foozleface thanks for providing us with the problem AND the fix. I will consider this issue fixed by now. Please re-open if that is not the case.
